README.md

```diff
@@ -165,6 +165,14 @@ explore('tests/data/foo.min.js', { output: { format: 'html' } }).then()
 
 See more at [wiki page][api wiki]
 
+## What might contribute to a generated file size
+
+In addition to mapped generated code a file may contain:
+
+- `sourceMappingURL` comment - A comment containing source map or referencing the file with source map. Represented by `[sourceMappingURL]` in explore result
+- Mapped code without source. It might be code generated by a bundler (e.g. webpack). Represented by `[no source]` in explore result
+- Unmapped code - code that is not referenced within the source map. Represented by `[unmapped]` in explore result.
```

Comment:
Suggest adding an example of how this might occur.

Reply:
Done